tests/rag_answers/test_invalid_json_retry.py

```diff
@@ -0,0 +1,141 @@
+from collections.abc import Sequence
+from typing import Callable
+from unittest.mock import AsyncMock
+
+import pytest
+from deepeval.models.llms.amazon_bedrock_model import AmazonBedrockModel
```

Comment:
I wonder if we can reference the subclass here of DeepEvalBaseLLM since nothing is specific for AmazonBedrockModel